tools/scripts/build.ps1

```diff
@@ -0,0 +1,15 @@
+choco install ant
+choco install vswhere
+choco install zip
+$env:WINSDK_DIR = (Get-ItemProperty -Path "HKLM:\SOFTWARE\Wow6432Node\Microsoft\Windows Kits\Installed Roots").KitsRoot10
+$env:VCINSTALLDIR = "$(vswhere -legacy -latest -property installationPath)\VC\Auxiliary\Build"
+$msvcToolsVer = Get-Content "$env:VCINSTALLDIR\Microsoft.VCToolsVersion.default.txt"
+if ([string]::IsNullOrWhitespace($msvcToolsVer)) {
+  # The MSVC tools version file can have txt *or* props extension.
+  $msvcToolsVer = Get-Content "$env:VCINSTALLDIR\Microsoft.VCToolsVersion.default.props"
+}
+$env:MSVC_VER = $msvcToolsVer
+$env:VS150COMNTOOLS = $env:VCINSTALLDIR
+$env:VSVARS32FILE = "$env:VCINSTALLDIR\vcvars32.bat"
+refreshenv
+.\gradlew all test -PCOMPILE_WEBKIT=false -PCONF=DebugNative --stacktrace -x :web:test --info
```

Comment:
I recommend adding `--no-daemon` which was present in the appveyor.yml file.

Reply:
For CI builds I wanted to use `--no-daemon`. For local builds it makes sense to use the daemon IMO. So in the latest commit we check if `APPVEYOR` environment variable is set. If so, use `--no-daemon`. Otherwise, don't. Is that okay?

Reply:
I broke it, let me fix it.

Reply:
Actually, strange, it seems that the downloading of OpenJDK 10 is what failed. Let me see.